owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -203,10 +214,27 @@ class FileListAdapter(
                         it.Filename.text = file.fileName
                         it.fileListSize.text = DisplayUtils.bytesToHumanReadable(file.length, context)
                         it.fileListLastMod.text = DisplayUtils.getRelativeTimestamp(context, file.modificationTimestamp)
-                        it.fileListPath.apply {
-                            text = file.remotePath
-                            isVisible = !fileListOption.isAllFiles()
+                        if (fileListOption.isAvailableOffline()) {
+                            it.fileListPath.apply {
+                                text = File(file.remotePath).parent
```

Comment:
There is a function at file to retrieve the parent path.

```suggestion
                                text = file.getParentRemotePath()
```

Reply:
Aaah nice! Didn't know about this 👍